content/Mobile_Dev/WebAuthn/IOS/Improve_Mobile_User_Experience.adoc

```diff
@@ -0,0 +1,174 @@
+
+In this section we extend the application improvements we made above to add guided prompts to help their user in beginning their WebAuthn journey. 
+
+==== Application demo
+The video below demonstrates our application which now includes guided prompts. These prompts provide step-by-step instructions for users to follow, greatly assisting users who aren’t familiar with WebAuthn.. We have provided guidance for both security keys, and platform authenticators. It’s also worth noting that for platform authenticators, we are providing guidance based on the assumed authenticator that exists on the device. In the example below we are showing the guidance for Face ID, which is often the authenticator found on iOS devices. 
+
+video::videos/mobile_ux_1.mp4[height=600]
+
+Figure 4 shows a still image of our example guidance for security keys
+
+image::images/mobile_ux_3.jpg[width=250]
+**Figure 4**
+
+Figure 5 shows a still image of our example guidance for Face ID.
+
+image::images/mobile_ux_4.jpg[width=250]
+**Figure 5**
+
+==== Implementing registration guidance
+This flow could be implemented using a simple pop up modal when the user clicks the ‘Add’ buttons on the security screen, rather than immediately invoking the WebAuthn ceremony. Our focus in this section will be implementing a helper to help you detect the user’s operating system to try and assume the platform authenticator.
+
+Figure 6 shows a code sample that includes a method that when invoked uses the browser’s user agent to determine the operating system. The OS is then mapped to a predefined list which includes the assumed platform authenticator.
+
+[source,javascript]
+----
+const platformEnums = {
+  ANDROID_BIOMETRICS: {
+    id: "ANDROID_BIOMETRICS",
+    platName: "Andorid Biometrics",
+  },
+  WINDOWS_HELLO: {
+    id: "WINDOWS_HELLO",
+    platName: "Windows Hello",
+  },
+  TOUCH_ID: {
+    id: "TOUCH_ID",
+    platName: "Touch ID",
+  },
+  FACE_ID: {
+    id: "FACE_ID",
+    platName: "Face ID",
+  },
+  DEFAULT: {
+    id: "DEFAULT",
+    platName: "this device",
+  },
+};
+
+const getPlatform = () => {
+  const { userAgent } = navigator;
+  if (userAgent.indexOf("Windows") !== -1) {
+    return platformEnums.WINDOWS_HELLO;
+  }
+  if (userAgent.indexOf("Android") !== -1) {
+    return platformEnums.ANDROID_BIOMETRICS;
+  }
+  if (
+    userAgent.indexOf("Macintosh") !== -1 ||
+    userAgent.indexOf("iPad") !== -1
+  ) {
+    return platformEnums.TOUCH_ID;
+  }
+  if (userAgent.indexOf("iPhone") !== -1) {
+    return platformEnums.FACE_ID;
+  }
+  return platformEnums.DEFAULT;
+};
+
+const DetectBrowser = { getPlatform, platformEnums };
+
+export default DetectBrowser;
+----
+
+It should be noted that this method isn’t always perfect. A user could manually change their user agent, leading to the incorrect user agent being assumed. This will not affect the usability, or ability for users to register with WebAuthn - Though there may be some confusion. 
```

Comment:
Consider using the Note notation in asciidocs here

Reply:
Changed in latest commit

I went through other areas of the docs and changed to note notation where appropriate